Automattic/wp-calypso
# Allow root AAAA records

#### Changes proposed in this Pull Request

* Allow AAAA records on the root domain

#### Testing instructions

Depends on D70067-code

Follow the test instructions on the back-end patch, but use Calypso to add/remove the records. Make sure that setting/deleting AAAA records works as expected.
